integrations/standalone/vite.config.ts

```diff
@@ -9,7 +9,12 @@ export default defineConfig({
     chunkSizeWarningLimit: 5000,
     rollupOptions: { input: { index: './index.html' } }
   },
-  server: { port: 3000 },
+  esbuild: {
+    supported: {
+      'top-level-await': true
+    }
```

Comment:
what is this?

Reply:
Without this, packaging will throw the following error in `config-editor-client/integrations/standalone/src/index.tsx`:
```
@axonivy/config-editor-standalone: Error: [vite:esbuild-transpile] Transform failed with 1 error:
@axonivy/config-editor-standalone: assets/index-!~{001}~.js:25422:15: ERROR: Top-level await is not available in the configured target environment ("chrome87", "edge88", "es2020", "firefox78", "safari14" + 2 overrides)
@axonivy/config-editor-standalone: 
@axonivy/config-editor-standalone: Top-level await is not available in the configured target environment ("chrome87", "edge88", "es2020", "firefox78", "safari14" + 2 overrides)
@axonivy/config-editor-standalone: 25420|  const file = URLParams.file();
@axonivy/config-editor-standalone: 25421|  const theme = URLParams.theme();
@axonivy/config-editor-standalone: 25422|  const client = await ClientJsonRpc.startWebSocketClient(server);
@axonivy/config-editor-standalone:    |                 ^
```
Is there a better way to handle this?

Reply:
As the other editors do it (where I assume you copied it ;)):
Put it into an async function which you then simply call.

```tsx
export async function start(): Promise<void> {
  const server = URLParams.webSocketBase();
  const app = URLParams.app();
  const pmv = URLParams.pmv();
  const file = URLParams.file();
  const theme = URLParams.theme();

  const client = await FormClientJsonRpc.startWebSocketClient(server);
  const queryClient = initQueryClient();

  createRoot(document.getElementById('root')!).render(
    <React.StrictMode>
      <ThemeProvider defaultTheme={theme}>
        <ClientContextProvider client={client}>
          <QueryProvider client={queryClient}>
            <App app={app} pmv={pmv} file={file} />
          </QueryProvider>
        </ClientContextProvider>
      </ThemeProvider>
    </React.StrictMode>
  );
}

start();
```